features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/apistore/scripts/apistore.js

```diff
@@ -97,19 +98,41 @@ var apistore = {};
         return deniedTiers;
     };
 
-
+    // TODO: this needs to be change
```

Comment:
Please do the necessary changes. if this is a change that cannot be done you need to create a JIRA and add proper details